lib/components/home/move_note_button.dart

```diff
@@ -153,8 +148,8 @@ class _MoveNoteDialogState extends State<_MoveNoteDialog> {
                 ],
               ),
             ),
-            if (newFileName != null && newFileName != fileName)
-              Text(t.home.moveNote.renamedTo(newName: newFileName ?? '?')),
+            if (newFileNames != null && newFileNames!.length == 1 && newFileNames![0] != fileNames[0])
+              Text(t.home.moveNote.renamedTo(newName: newFileNames![0])),
```

Comment:
I think this could be improved to support multiple files, something like this:
```dart
if (newFileNames != null)
  for (int i = 0; i < fileNames.length; ++i)
    if (fileNames[i] != newFileNames![i])
      Text(t.home.moveNote.renamedTo(newName: newFileNames![i])),
```

But we could change the text to be like this if we have more than one changed file name (for only one changed file name, the old one is better)
```
Some files will be renamed when moved into this folder:
- somefile (2)
- notes (3)
```